packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -102,11 +111,14 @@ export class GroupedListBase extends React.Component<IGroupedListProps, IGrouped
 
     const { version } = listProps;
 
+    const { shouldEnterInnerZone = this._isInnerZoneKeystroke } = focusZoneProps;
+
     return (
       <FocusZone
-        isInnerZoneKeystroke={this._isInnerZoneKeystroke}
+        {...focusZoneProps}
```

Comment:
i think we should only override `focusZoneProps` with `direction` and `className` here